recipe/build.sh

```diff
@@ -1,27 +1,24 @@
 #!/bin/bash
 
+set -eo pipefail
+
+export PKG_CONFIG_PATH="${BUILD_PREFIX}/bin/pkg-config:${BUILD_PREFIX}/lib/pkgconfig:${PREFIX}/bin/pkg-config:${PREFIX}/lib/pkg-config"
+export PKG_CONFIG_EXECUTABLE=${PREFIX}/bin/pkg-config
+
 declare -a CMAKE_PLATFORM_FLAGS
-if [[ ${HOST} =~ .*linux.* ]]; then
-    CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
-fi
 
 mkdir build_shared && cd $_
-cmake \
+cmake -G "Ninja" \
+    ${CMAKE_ARGS} \
+    -DALLOW_IN_SOURCE_BUILD=ON \
     -DCMAKE_PREFIX_PATH=$PREFIX \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
-    -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=False \
-    ${CMAKE_PLATFORM_FLAGS[@]} ..
-make
-make install
-cd ..
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
+    -DJAS_ENABLE_LIBHEIF=OFF \
+    -DJAS_ENABLE_DOC=OFF \
+    -DJAS_PACKAGING=ON \
+    "${CMAKE_PLATFORM_FLAGS[@]}" ..
 
-mkdir build_static && cd $_
-cmake \
-    -DJAS_ENABLE_SHARED=OFF \
-    -DCMAKE_PREFIX_PATH=$PREFIX \
-    -DCMAKE_INSTALL_PREFIX=$PREFIX \
-    -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=False \
-    ${CMAKE_PLATFORM_FLAGS[@]} ..
-make
-make install
+cmake --build . --config Release --target=install --parallel ${CPU_COUNT}
+ctest --output-on-failure || true
```

Comment:
what happens if you remove || true? can failing tests be explicitly skipped with reasons?

Reply:
There's `option(JAS_ENABLE_CXX "Enable C++ code (for testing)." OFF)`, dumping this here in case it's relevant

Reply:
Removed `|| true`